.circleci/config.yml

```diff
@@ -226,24 +248,122 @@ commands:
           working_directory: << parameters.dir >>
           command: docker save << parameters.image >> > image.tar
 
-  push_docker_image:
+  load_docker_image:
     parameters:
-      image:
-        type: string
       dir:
         type: string
     steps:
       - run:
           name: Load docker image
           working_directory: << parameters.dir >>
           command: docker load -i image.tar
+
+  push_docker_image:
+    parameters:
+      image:
+        type: string
+      dir:
+        type: string
+    steps:
+      - load_docker_image:
+          dir: << parameters.dir >>
       - run:
           name: Push docker image
           command: |
             aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin 518207624130.dkr.ecr.eu-west-1.amazonaws.com
             docker tag << parameters.image >> 518207624130.dkr.ecr.eu-west-1.amazonaws.com/<< parameters.image >>:${CIRCLE_SHA1}
             docker push 518207624130.dkr.ecr.eu-west-1.amazonaws.com/<< parameters.image >>:${CIRCLE_SHA1}
 
+  e2e_build:
+    steps:
+      - *restore_repo
+      - setup_remote_docker:
+          version: *remote_docker_version
+          docker_layer_caching: false
+      - run:
+          name: "Setup E2E_IMAGE and store date"
+          command: |
+            echo 'export E2E_IMAGE="518207624130.dkr.ecr.eu-west-1.amazonaws.com/evaka/e2e-playwright:latest"' >> $BASH_ENV
+            date +%F > date
+      - restore_cache:
+          name: Restore Docker image from cache
+          key: playwright-daily-v1-{{ checksum "date"}}
+      - run:
+          name: Build E2E image
+          working_directory: *workspace_compose
+          command: |
+            docker load -i "*workspace_root/e2e-playwright.tar" || true
```

Comment:
This isn't working: `open *workspace_root/e2e-playwright.tar: no such file or directory`